pyuvdata/uvdata/uvdata.py

```diff
@@ -3839,6 +3846,10 @@ def sum_vis(
         strict_uvw_antpos_check : bool
             Option to raise an error rather than a warning if the check that
             uvws match antenna positions does not pass.
+        override_params : array_like of strings
+            List of object UVParameters to omit from compatibility check. Overridden
+            parameters will not be compared between the objects, and the values
+            for these parameters will be taken from the first object.
```

Comment:
It seems like this will be used so rarely that it's not worth providing this extra flexibility.  It's only an option for a black-belt user anyway and they'll be able to come up with their own workaround (e.g. changing the parameter value in the first object before doing the sum).